docs/en/UI/Angular/Form-Validation.md

```diff
@@ -0,0 +1,144 @@
+# Form Validation
+
+Reactive forms in ABP Angular UI are validated by [ngx-validate](https://www.npmjs.com/package/@ngx-validate/core) and helper texts are shown automatically based on validation rules and error blueprints. You do not have to add any elements or components to your templates. The library handles that for you. Here is how the experience is:
+
+<img alt="The ngx-validate library validates an Angular reactive form and an error text appears under each wrong input based on the validation rule and the error blueprint." src="./images/form-validation---error-display-user-experience.gif" width="990px" style="max-width:100%">
```

Comment:
change image link to  ![form-validation-error-display-user-experience](./images/form-validation---error-display-user-experience.gif)

Reply:
Why do you think that? It works.

Reply:
Hi @rqx110,

The first part (left side) of a Markdown image is the alternate text and it is important for accessibility. The screen readers will read this text out loud to the user, so it must be a meaningful explanation of the image.